x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -146,21 +150,18 @@ describe('policy table', () => {
     snapshot(namesText(rendered));
   });
   test('should sort when name header is clicked', () => {
-    testSort('name');
-  });
-  test('should sort when version header is clicked', () => {
-    testSort('version');
+    testSort('name_0');
   });
   test('should sort when modified date header is clicked', () => {
-    testSort('modifiedDate');
+    testSort('modifiedDate_3');
```

Comment:
nit: are these `data-test-subj` attributes generated automatically by EUI? I found it a little hard to parse initially with the `index` included. If it's possible to pass our own `data-test-subj` I think that might make this more clear.

Reply:
This I have to leave as is for now, because using `EuiInMemoryTable` prevents us from adding custom `data-test-subj` to the table header buttons.